templates/node/src/Payload.ts.twig

```diff
@@ -0,0 +1,58 @@
+import { readFileSync } from "fs";
+import { File } from "node-fetch-native-with-agent";
+import { basename } from "path";
+
+export class Payload extends File {
+  private data: Buffer;
+  private fileName: string;
+
+  constructor(data: Buffer, name?: string) {
+    super([data], name || 'unnamed');
+    this.data = data;
+    this.fileName = name || 'unnamed';
```

Comment:
I would avoid doing "unnamed". It should allow null value, and then in endpoints, throw exception if name is needed and is not provided - like createStorage

Reply:
I need to do it like this because of strict type checking in ts